python/sunbeam/schedule.py

```diff
@@ -16,6 +16,22 @@ def wells(self):
     def group(self, name):
         return Group(self._group(name), self)
 
+    def group_tree(self, timestep):
+        return GroupTree(self._group_tree(timestep))
+
+    def gtree(self, timestep):
+        '''Fuction to return python struct of group topologi for given timestep'''
```

Comment:
> `'''Fuction to return python struct of group topologi for given timestep'''`
Please use `"""` for docstrings (check out PEP-8), use spell checker ("Fuction"?), and write useful docstrings.